### PR TITLE
adapted for 18 groups

### PR DESCRIPTION
**Pull request by beccaelenzil**

### CODE REVIEW

**Comment by beccaelenzil** [comment]
I wanted to note that the instructions references a merge conflicts lesson that I don't think we do anymore. Also, there's this resource for intro-ing: https://docs.google.com/presentation/d/15OFyyoNJevzk7IpdZCP6VwMLju0MVS2pv0qeETLOa6Q/edit#slide=id.g204c54e54e_0_58

File: merge-conflict-remote.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+### Building the recipe
+
+Once everyone in your group has completed the baseline setup instructions above you're ready to start building the recipe together.
+
+#### General advice
+
+There are a few things to look out for when working through this activity:
+
+- You may want to review [Intro to Git](https://learn-2.galvanize.com/cohorts/2498/blocks/1032/content_files/intro-to-git/collaborating.md) and  [Branches in Git](https://learn-2.galvanize.com/cohorts/2498/blocks/1441/content_files/branches-in-git/branches.md).
+- You can review the [slides](https://docs.google.com/presentation/d/15OFyyoNJevzk7IpdZCP6VwMLju0MVS2pv0qeETLOa6Q/edit?usp=sharing)
```
Comment (audreyandoy) [inline merge-conflict-remote.md]: Move this to `readme` resources list

File: merge-conflict-remote.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+1. **Split into groups**: The class will be divided into groups by Homeroom. These groups will be your groups for the project next week. 
+   - Breakout Rooms 1, 7, 13: Aardvark
+   - Breakout Room 2, 8, 14: Tarsier
+   - Breakout Room 3, 9, 15: Gecko
+   - Breakout Room 4, 10, 16: Sloth
+   - Breakout Room 5, 11, 17: Lionfish
+   - Breakout Room 6, 12, 18: Pangolin
+ 
+1. **Fork the activity repo**: One person in your group should fork the [carnival-recipes](https://github.com/AdaGold/carnival-recipes) repo and then add everyone else as a collaborator.
```
Comment (audreyandoy) [inline merge-conflict-remote.md:29]: Add instructions to add collaborator

Comment (audreyandoy) [reply merge-conflict-remote.md:29]: And how accept collaborator invitations

File: merge-conflict-remote.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+- If you find that the result of one of your merges is missing critical ingredients or any numbered steps, you should go back and double check that you didn't lose any lines while resolving a merge conflict.
+- Your recipe scraps may have duplicate lines. We've added these in to more accurately portray the full range of potential merge conflicts. **Important**: you should get rid of any duplicate lines while merging.
+- There's no "right" way to do this activity, all of the strategies listed below are valid ways to build the recipe file.
+- Have fun!
+
+#### Recipe building process
+
+Because each member of your group now has a different version of the recipe file in their local repository, building the complete recipe by merging those versions together will result in merge conflicts. To handle those merge conflicts sensibly your group should use the following process to construct the final recipe file:
+
+1. Each person on a team will create a branch with `git checkout -b BRANCH-NAME`, selecting their own branch name based off the recipe scrap they have, and commit the changes they have made to the recipe.
```
Comment (audreyandoy) [inline merge-conflict-remote.md:45]: also include `git switch`

File: merge-conflict-remote.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+1. **Split into groups**: The class will be divided into groups by Homeroom. These groups will be your groups for the project next week. 
+   - Breakout Rooms 1, 7, 13: Aardvark
+   - Breakout Room 2, 8, 14: Tarsier
+   - Breakout Room 3, 9, 15: Gecko
+   - Breakout Room 4, 10, 16: Sloth
+   - Breakout Room 5, 11, 17: Lionfish
+   - Breakout Room 6, 12, 18: Pangolin
+ 
+1. **Fork the activity repo**: One person in your group should fork the [carnival-recipes](https://github.com/AdaGold/carnival-recipes) repo and then add everyone else as a collaborator.
+1. **Clone the forked repo**: Everyone in your group should clone the forked repository.
```
Comment (audreyandoy) [inline merge-conflict-remote.md:32]: Show students `git remote` to confirm that they cloned their partner's repo NOT ada-c15 repo

File: merge-conflict-remote.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+1. **Find your recipe scrap**: Each group is assigned six(6) or seven(7) recipe scraps that should all be part of the same recipe. You can split these scraps up amongst your group however you wish. The scraps assigned to each group are listed in the carnival-recipes repo.
+1. **Add your recipe scrap**: Copy the contents of your recipe scrap into the recipe file and then add and commit your changes to the recipe file. After completing this step each person in your group should have **different** content within the recipe file in their local repository.
```
Comment (audreyandoy) [inline merge-conflict-remote.md:35]: probably talk about adding a message when merging pRs

Comment (audreyandoy) [reply merge-conflict-remote.md:35]: include instructions about switching base repository to their partners

Comment (audreyandoy) [reply merge-conflict-remote.md:35]: MERGE CONFLICT RESOLVING:

Person who causes conflict has to fix by doing:

- `git pull origin master` 
-  accept both changes (likely will have to explain current changes, incoming changes, both changes)
-  rearrange recipe
- then push onto branch

**Review by audreyandoy — COMMENTED** [review]
Some notes from our rehearsal

File: merge-conflict-remote.md (hunk 5 above)
Comment (beccaelenzil) [reply merge-conflict-remote.md:35]: I put this content on the step below about making pull requests and resolving merge conflicts. What do you think?

File: merge-conflict-remote.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+1. Each person on a team will create a branch with `git checkout -b BRANCH-NAME`, selecting their own branch name based off the recipe scrap they have, and commit the changes they have made to the recipe.
+1. Each team member will push their changes up to github with `git push origin BRANCH-NAME`.
+1. Each team member will open a **pull request** trying to merge their branch on github into `master`.  **Be very careful to make the PR against your forked repository and NOT AdaGold.**
+    - To make the PR against your team members forked respository and not AdaGold, we will need to change the base repository
+    - If there is a report of a merge conflict you will need to:
+       - Pull the current state of master into **their feature branch** with `git pull origin master`
+       - in VS code select **accept both changes**
+       - Resolve the merge conflicts by rearranging the recipe and commit the result
+       - Push the result up to git hub with `git push origin BRANCH-NAME`
```
Comment (audreyandoy) [inline merge-conflict-remote.md]:
```suggestion
       - Push the result up to github with `git push origin BRANCH-NAME`
```

File: merge-conflict-remote.md (hunk 5 above)
Comment (audreyandoy) [reply merge-conflict-remote.md:35]: LGTM 👍

**Review by audreyandoy — APPROVED** [review]
LGTM 👍

**Pull request merged**